Documentation/teaching/info/vm.rst

```diff
@@ -4,118 +4,118 @@
 Virtual Machine Setup
 =====================
 
-Exercises are designed to run on a qemu based virtual machine. In
-order to run the virtual machine you will need following packages:
+Practice work is to designed to run on a QEMU based virtual machine. In order
+to run and use the virtual machine the following packages are required:
 
-* build-essential
-* qemu-system-x86
-* qemu-system-arm
-* kvm
-* python3
+* ``build-essential``
+* ``qemu-system-x86``
+* ``qemu-system-arm``
+* ``kvm``
+* ``python3``
+* ``minicom``
```

Comment:
Cam we add the 32 bit libraries, maybe as a note to mention they are needed for some of the labs.

Reply:
@Sergiu121 , what are the specific 32 packages that you installed in the Docker environment?

Reply:
I just used gcc-multilib. It's all I needed.

Reply:
OK, I added `gcc-multilib`.